include/envoy/access_log/access_log.h

```diff
@@ -84,5 +84,20 @@ class Formatter {
 
 typedef std::unique_ptr<Formatter> FormatterPtr;
 
+/**
+ * Interface for access log provider
```

Comment:
Can you put this explanation that you have of `Formatter` vs `FormatterProvider` in the PR commit message here as well?

Reply:
Agreed
- [ ] Add FormatterProvider explanation